proposal/prefixed-parent-selector.md

```diff
@@ -0,0 +1,58 @@
+# Prefixed Parent Selector: Draft 2
+
+[Issue](https://github.com/sass/sass/issues/1425)
+
+## Background
+
+When transitioning from Less or Stylus, or just learning Sass, nesting is common.
+
+Everything works just as expected until a `&` is in the beginning of a nested selector.
+
+`"&" may only used at the beginning of a compound selector.` 
+
+Considering that there's been quite a few issues regarding this, and that they keep comming up, 
+and that people expect this to work, I think this would be a valuable addition to Sass.
+
+There's a work-around ([with some limitations](https://github.com/sass/sass/issues/1425#issuecomment-404462836)),
+but it's subpar. It might scare of beginners and/or cause unnecessary headache.
```

Comment:
The role of the "background" section is to provide a single location that combines all the pre-existing discussion around the issue. As such, please explain the workaround in detail here, rather than requiring users to read through comments in an issue thread.

Reply:
Attempted to address this in: https://github.com/sass/sass/pull/2723/commits/7631df1dfd0e4b5ca6b134834d9e1d9ded17c7d9